tensorflow/core/api_def/base_api/api_def_StringLower.pbtxt

```diff
@@ -1,4 +1,7 @@
 op {
   graph_op_name: "StringLower"
   description: "Converts each string in the input Tensor to lowercase."
```

Comment:
The format of description in here allows for only a single line. For more lines in the description you have to use a here string:

```
description <<< END
Text of the description
It can have multiple lines
It will end when END shows up on a line by itself
END
```

Reply:
Thank you